meteor/client/ui/RundownView/RundownRightHandControls.tsx

```diff
@@ -108,19 +106,22 @@ export function RundownRightHandControls(props: Readonly<IProps>): JSX.Element {
 		setSwitchboardOpen(false)
 	}
 
-	const availableRouteSets = Object.entries<StudioRouteSet>(props.studioRouteSets).filter(
-		([_id, routeSet]) => routeSet.behavior !== StudioRouteBehavior.HIDDEN
-	)
+	const availableRouteSets = getAllCurrentItemsFromOverrides(props.studioRouteSets, null)
```

Comment:
This should change following my comment on `UIStudio`, but calling `getAllCurrentItemsFromOverrides` like this in the render is not a good idea. This is quite a heavy operation, as it does a deep clone of the route sets.
Doing this on every render of this component is costly, it would be a good idea to wrap it in `useMemo` so that it is only recomputed when `props.studioRouteSets` actually changes.

Reply:
This will probably be reverted when changing the UI->RouteSet interface